### PR TITLE
fuzz/lnwire: minor touch-ups, remove MaxPayloadLength

### PR DESCRIPTION
**Pull request by Crypt-iQ**
With this pull, the `fuzz/lnwire` tests build and run without crashing.

### CODE REVIEW

**Comment by Crypt-iQ** [comment]
@bhandras @yyforyongyu feel free to remove yourself if you don't want to review

**Review by bhandras — APPROVED** [review]
LGTM :+1:

File: fuzz/lnwire/fuzz_utils.go [hunk 1]
```diff
@@ -24,25 +24,18 @@ func prefixWithMsgType(data []byte, prefix lnwire.MessageType) []byte {
...
-	// Make sure byte array length (excluding 2 bytes for message type) is
-	// less than max payload size for the wire message. We check this because
-	// otherwise `go-fuzz` will keep creating inputs that crash on ReadMessage
-	// due to a large message size.
-	payloadLen := uint32(len(data)) - 2
-	if payloadLen > emptyMsg.MaxPayloadLength(0) {
-		// Ignore this input - max payload constraint violated.
+	// Check that the created message is not greater than the maximum
+	// message size.
+	if len(data) > lnwire.MaxMessagePayload {
```
Comment (yyforyongyu) [inline fuzz/lnwire/fuzz_utils.go:33]: `MaxMessagePayload` is a duplicate param of `MaxSliceLength`, and it's removed in following commit.

**Review by yyforyongyu — COMMENTED** [review]
Changes look good, just a few comments.

File: fuzz/lnwire/fuzz_utils.go (hunk 1 above)
Comment (Crypt-iQ) [reply fuzz/lnwire/fuzz_utils.go:33]: There's only one commit - do you mean #4884

Comment (yyforyongyu) [reply fuzz/lnwire/fuzz_utils.go:33]: Yeah that's right. Plus I think we should stick to the `MaxSliceLength` and `MaxMsgBody` split.

**Review by Roasbeef — APPROVED** [review]
LGTM 📯

Just needs a rebase and an update to the release notes folder now.

File: fuzz/lnwire/fuzz_utils.go (hunk 1 above)
Comment (Crypt-iQ) [reply fuzz/lnwire/fuzz_utils.go]: addressed

**Comment by Crypt-iQ** [comment]
updated release notes

**Pull request merged**